pkg/node/utils.go

```diff
@@ -174,7 +173,8 @@ func FindDrives(ctx context.Context, nodeID string, procfs string) ([]*direct_cs
 			}
 		}
 		v.OwnerNode = nodeID
-		driveName := uuid.NewUUID().String()
+		objName := strings.Join([]string{nodeID, v.Path}, "-")
+		driveName := fmt.Sprintf("%x", md5.Sum([]byte(objName)))
```

Comment:
instead of md5 use sha256 @Praveenrajmani to avoid any conflicts `sha256-simd` package can be used here. 

Also not sure why the variable name is `objName` isn't it `driveName` ?

Reply:
Will change @harshavardhana